avm/utilities/pipelines/staticValidation/compliance/helper/helper.psm1

```diff
@@ -10,6 +10,7 @@ $repoRootPath = (Get-Item $PSScriptRoot).Parent.Parent.Parent.Parent.Parent.Pare
 . (Join-Path $repoRootPath 'avm' 'utilities' 'pipelines' 'sharedScripts' 'helper' 'Get-IsParameterRequired.ps1')
 . (Join-Path $repoRootPath 'avm' 'utilities' 'pipelines' 'sharedScripts' 'helper' 'ConvertTo-OrderedHashtable.ps1')
 . (Join-Path $repoRootPath 'avm' 'utilities' 'pipelines' 'sharedScripts' 'helper' 'Get-CrossReferencedModuleList.ps1')
+. (Join-Path $repoRootPath 'avm' 'utilities' 'pipelines' 'publish' 'helper' 'Get-BRMRepositoryName.ps1')
```

Comment:
Is this needed?

Reply:
It is. The function is used in the test and hence must be loaded ahead of time

Reply:
Ideally this should be moved to the sharedScripts folder then. Not for this PR, but should we keep track of those improvements?

Reply:
Will note it down